server/service/src/sync/translation_and_integration.rs

```diff
@@ -34,19 +34,15 @@ impl<'a> TranslationAndIntegration<'a> {
         }
     }
 
+    // Go through each translator, adding translations to result, if no translators matched return None
     fn translate_sync_record(
         &self,
         sync_record: &SyncBufferRow,
         translators: &SyncTanslators,
     ) -> Result<Option<IntegrationRecords>, anyhow::Error> {
-        let mut translators_iter = translators.iter();
```

Comment:
don't need iter now, the whole point of loop like it was, is to early return with break and result